.env.example

```
@@ -0,0 +1,8 @@
+DB_USER=*Entrer nom utilisateur ici*
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)